etk-asm/src/parse/asm.pest

```diff
@@ -35,11 +35,11 @@ half_word_size = @{ ("1" ~ '0'..'6') | '1'..'9' }
 // instruction macros //
 ////////////////////////
 instruction_macro_definition = { "%macro" ~ function_declaration ~ NEWLINE* ~ (instruction_macro_stmt ~ NEWLINE)* ~ "%end" }
-instruction_macro_stmt = _{ label_definition | builtin | instruction_macro | push | op }
+instruction_macro_stmt = _{ label_definition | "%" ~ push_macro | local_macro | push | op }
```

Comment:
Why do we treat `push_macro` separately from user defined macros? I think they have the same syntax.

Reply:
Hello ser @SamWilsn - any further thoughts here?